preview/launcher/powershell-wrapper

```diff
@@ -43,6 +50,9 @@ TestDistro(){
     DIST=`lowercase $DIST`
 
     # Tests for unsupported distrobutions go here
+    if [ "${DIST}" = "manjaro" ] ; then
```

Comment:
technically arch is community supported... would that fall under unsupported in this case?

What about the other weird distros that snap supports:
![image](https://user-images.githubusercontent.com/2644648/42414199-e03922dc-81e4-11e8-98c6-122a5125c7f8.png)
Solus, yocto, openwrt, etc.

Reply:
I need to know how to detect them.  I'll update the PR for Arch as well

Reply:
Should this be `DistroBasedOn` not `DIST`?

Reply:
or line 55 be `DIST`

Reply:
Technically another distro could base theirs on manjaro and not change the factor I use to determine DistroBasedOn.

Reply:
Look up on line 23 for the amazon linux vs redhat code.